src/bloom.cpp

```diff
@@ -87,14 +87,15 @@ void CBloomFilter::insert(const vector<unsigned char> &vKey)
     isEmpty = false;
 }
 
-void CBloomFilter::insert(const COutPoint &outpoint)
+static std::vector<unsigned char> ToVector(const COutPoint &outpoint)
```

Comment:
wonder if changes to `bloom.{cpp,h}` could be resubmitted on a separated PR. no strong opinion thou.

Reply:
This has it's own commit, but I don't think it's significant enough to deserve it's own PR. I'll split it into multiple PRs if you insist though :-)

Reply:
No need to splut, we can keep it here.